sm_property_tree/include/sm/BoostPropertyTree.hpp

```diff
@@ -87,14 +87,23 @@ namespace sm {
     static bool isHumanReadableInputOutput() {
       return humanReadableInputOutput;
     }
+    static bool isIgnoreComments() {
+      return ignoreComments;
+    }
 
     static void setHumanReadableInputOutput(bool humanReadableInputOutput) {
       BoostPropertyTree::humanReadableInputOutput = humanReadableInputOutput;
     }
+    static void setIgnoreComments(bool ignoreComments) {
+      BoostPropertyTree::ignoreComments = ignoreComments;
+    }
+    
+    static int getXmlReadOptions();
```

Comment:
Ah no, I was wrong. `BoostPropertyTreeImplementation` is not a child of `BoostPropertyTree`. One could make it a friend though?